src/main/java/com/acrolinx/sidebar/pojo/settings/CheckOptions.java

```diff
@@ -12,20 +12,20 @@ public class CheckOptions
 {
     private final InputFormat inputFormat;
     private final RequestDescription requestDescription;
-    private final DocumentSelection documentSelection;
+    private final DocumentSelection selection;
```

Comment:
another way to fix this problem would be to add a

`@SerializedName("selection")`

annotation to this field, then we could name it whatever we want (e.g. "documentSelection") but Gson would serialize it as "selection" in the JSON string.

Not sure which solution I prefer, what do you think?

Reply:
Both work for me. I have no issue using `@SerializedName("selection")`.